### ISSUE TITLE
Telemetry dialog opens under main window and blocks interaction

### ISSUE THREAD

**Issue by tigert** [containers/podman-desktop#1887]
### Is your enhancement related to a problem? Please describe

Saw a tweet by  @mairin  and found this app, it is looking nice!

However when I installed the flatpak on my Fedora, I wondered why the purple button to get past the initial welcome screen did not appear to work. 

Turns out the UI was blocked by the dialog requesting permission to enable telemetry. And the dialog was hidden behind the main window.

### Describe the solution you'd like

So the dialog should definitely me modal and open on top of everything else, but I think that would not be too nice either, as it breaks the first use flow quite abruptly.

Ideally, I think, it should open after a bit of use, to not interrupt the user experience when you are trying to figure out how the app works. as it seems very smooth user experience otherwise. Maybe the question for telemetry should come after a bit of use?

"Looks like you are enjoying Podman Desktop, would you like to improve it by sharing anonymised usage data...?"

### Describe alternatives you've considered

Making it just modal would solve the confusion of frozen UI, but it would still interrupt your user experience.

**Comment by benoitf** [comment]
closing as a duplicate of https://github.com/containers/podman-desktop/issues/664
we're working on using the welcome screen instead of this 'under main window' popup

**Comment by mairin** [comment]
Thanks for the report, Tuomas! We just worked on the design of the fix for
this yesterday :-D If you see any other UX issues let us know, I would love
your perspective!!

**Comment by tigert** [comment]
Yeah, noticed! Sorry for the duplicate! :)

**Comment by benoitf** [comment]
It's better to have duplicates rather than no feedback, so thanks for the report.
